crates/ruff_linter/src/rules/airflow/rules/removal_in_3.rs

```diff
@@ -12,6 +15,7 @@ enum Replacement {
     None,
     Name(&'static str),
     Message(&'static str),
+    ContextRemovalMessage(&'static str),
```

Comment:
Simply `Context` or `ContextItem` is proabaly enough. I see the mesage provided here is always `is removed in Airflow 3.0.`. Do we plan to add other messages? If not, we can probably just hard-code it in Violation instead.

Reply:
I have hard-coded it in Violation now: https://github.com/astral-sh/ruff/pull/15144/commits/72dcad72e3f0ea0e6c008341bf5f6ec615d7fdb4